addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java

```diff
@@ -249,8 +252,6 @@ private enum RDBMS {
                 "Sybase",
                 Tech.Sybase,
                 "\\Qcom.sybase.jdbc\\E",
-                "\\Qcom.sybase.jdbc2.jdbc\\E",
-                "\\Qcom.sybase.jdbc3.jdbc\\E",
                 "\\Qnet.sourceforge.jtds.jdbc\\E" // see also Microsoft SQL Server. could be either!
```

Comment:
Removed these as they are not needed - the line above already matches them. Funny what you find when you add tests 😉